src/DiffOp/Diff/Diff.php

```diff
@@ -12,6 +12,8 @@
  * Base class for diffs. Diffs are collections of DiffOp objects,
  * and are themselves DiffOp objects as well.
  *
+ * @SuppressWarnings(PHPMD.ExcessiveClassComplexity)
+ *
```

Comment:
I do not think it's easy to get rid of, not that it is worth doing so (considering cost-benefit). It's too bad we ended up with this class as such, but it's there now. Increasing the rule to allow for it then won't catch new such introductions, which should not happen.